### PR TITLE
VB-2969 Remove unused HMPPS Auth code

### PR DESCRIPTION
**Pull request by tpmcgowan**
* Remove unused HMPPS Auth code, following implementation of GOV.UK One Login
* Add Orchestration API to the health check
* Add an integration test for the API call test page to check that getting tokens for API calls from HMPPS Auth (which is still needed) is mocked and working

### CODE REVIEW

File: server/config.ts [hunk 1]
```diff
@@ -49,14 +49,11 @@ export default {
   apis: {
     hmppsAuth: {
       url: get('HMPPS_AUTH_URL', 'http://localhost:9090/auth', requiredInProduction),
-      externalUrl: get('HMPPS_AUTH_EXTERNAL_URL', get('HMPPS_AUTH_URL', 'http://localhost:9090/auth')),
       timeout: {
         response: Number(get('HMPPS_AUTH_TIMEOUT_RESPONSE', 10000)),
         deadline: Number(get('HMPPS_AUTH_TIMEOUT_DEADLINE', 10000)),
       },
       agent: new AgentConfig(Number(get('HMPPS_AUTH_TIMEOUT_RESPONSE', 10000))),
-      apiClientId: get('API_CLIENT_ID', 'clientid', requiredInProduction),
```
Comment (tpmcgowan) [inline server/config.ts:58]: Removing everything related to `API_CLIENT` as we only need `SYSTEM_CLIENT` (for API calls)

File: server/config.ts [hunk 2]
```diff
@@ -65,15 +62,6 @@ export default {
       clientId: get('GOVUK_ONE_LOGIN_CLIENT_ID', 'clientId', requiredInProduction),
       privateKey: get('GOVUK_ONE_LOGIN_PRIVATE_KEY', 'privateKey', requiredInProduction),
     },
-    tokenVerification: {
```
Comment (tpmcgowan) [inline server/config.ts:68]: All token verification stuff removed as this was only used for verifying the _users_ token with HMPPS Auth when that was how they logged in.

File: server/services/healthCheck.ts [hunk 3]
```diff
@@ -49,16 +49,8 @@ function gatherCheckInfo(aggregateStatus: Record<string, unknown>, currentStatus
...
-  ...(config.apis.tokenVerification.enabled
-    ? [
-        service(
-          'tokenVerification',
-          `${config.apis.tokenVerification.url}/health/ping`,
-          config.apis.tokenVerification.agent,
-        ),
-      ]
-    : []),
+const apiChecks: HealthCheckService[] = [
```
Comment (tpmcgowan) [inline server/services/healthCheck.ts:52]: I know the health check in the template has changed a bit recently; will do a separate PR to pull in template changes at some point...

File: server/data/hmppsAuthClient.ts [hunk 4]
```diff
@@ -10,15 +10,14 @@ import RestClient from './restClient'
 const timeoutSpec = config.apis.hmppsAuth.timeout
 const hmppsAuthUrl = config.apis.hmppsAuth.url
 
-function getSystemClientTokenFromHmppsAuth(username?: string): Promise<superagent.Response> {
+function getSystemClientTokenFromHmppsAuth(): Promise<superagent.Response> {
```
Comment (tpmcgowan) [inline server/data/hmppsAuthClient.ts:13]: Removing support for getting system client tokens with a `username` because not applicable with GOV.UK One Login.

File: integration_tests/mockApis/orchestration.ts
Comment (tpmcgowan) [inline integration_tests/mockApis/orchestration.ts:1]: GitHub seems to think I've renamed and changed this file - I have in fact deleted one and added another 🙄

File: integration_tests/mockApis/hmppsAuth.ts
Comment (tpmcgowan) [inline integration_tests/mockApis/hmppsAuth.ts:1]: Minimal HMPPS Auth wiremock stub still needed for getting system client token for API calls

File: feature.env [hunk 5]
```diff
@@ -1,10 +1,8 @@
...
 HMPPS_AUTH_URL=http://localhost:9091/auth
-TOKEN_VERIFICATION_API_URL=http://localhost:9091/verification
-TOKEN_VERIFICATION_ENABLED=true
 NODE_ENV=development
-API_CLIENT_ID=clientid
-API_CLIENT_SECRET=clientsecret
 SYSTEM_CLIENT_ID=clientid
 SYSTEM_CLIENT_SECRET=clientsecret
+
+ORCHESTRATION_API_URL=http://localhost:9091/orchestration
```
Comment (psoleckimoj) [inline feature.env:8]: Missing EOF newline

**Review by psoleckimoj — DISMISSED** [review]
LGTM!

**Review by psoleckimoj — APPROVED** [review]
LGTM!
